src/poggit/ci/builder/DefaultProjectBuilder.php

```diff
@@ -170,7 +171,8 @@ protected function build(Phar $phar, RepoZipball $zipball, WebhookProjectModel $
             $localName = $out . substr($file, strlen($in));
             $phar->addFromString($localName, $contents = $reader());
             if(Lang::startsWith($localName, "src/") and Lang::endsWith(strtolower($localName), ".php")) {
-                $this->lintPhpFile($result, $localName, $contents, $localName === $mainClassFile, ($doLint ? $project->manifest["lint"] : null));
+                $this->lintPhpFile($result, $localName, $contents, $localName === $mainClassFile,
+                    $pluginManifest["src-namespace-prefix"]??"", ($doLint ? $project->manifest["lint"] : null));
```

Comment:
src-namespace-prefix should be ignored if we're not on PM4.

Also, use of src-namespace-prefix should be a lint error on PM3 API plugins.

Reply:
API Parsing is not one of poggit's strong suite I'll see.

Defo haven't got time to be messing with new lints for api specific plugins rn (same could be said about OS field < 3.12.0)